### ISSUE TITLE
Netlify CMS v3 Hacktoberfest

### ISSUE THREAD

**Issue by DLevai94** [netlify/netlify-cms#4389]
Hey y'all,

I thought [Hacktoberfest](https://hacktoberfest.digitalocean.com/) would be an awesome opportunity to contribute to the new version of Netlify CMS.

This one: https://github.com/netlify/netlify-cms/issues/2557
Dedicated branch: https://github.com/netlify/netlify-cms/tree/future/v3
Project: https://github.com/netlify/netlify-cms/projects/9

I'd be pretty happy to help develop it and I think there're other Netlify CMS users who are thinking the same. 

I'm thinking about it would be great if we can identify the next steps in this process, create some issues for it, and a basic "How to contribute" guide to speed up things and make it easier to get started.

Maybe @austincondiff as the main head behind this could help us.

What do you think? How can we get started organizing this and development?

**Comment by JoseRFelix** [comment]
Definitely would love to help with this! Is there an issue you can recommend me to get started?

**Comment by Vlad412** [comment]
Hello !
Same as @JoseRFelix

**Comment by KoljaTM** [comment]
As an fyi https://hacktoberfest.digitalocean.com/hacktoberfest-update
tl;dr: In order to be eligible for hacktoberfest, the repo must have a "hacktoberfest" topic. This was added to avoid spam PRs.

**Comment by DLevai94** [comment]
@KoljaTM that's why I created this issue, to draw the maintainers' attention, hoping, they'll find this a good idea too.

**Comment by erezrokah** [comment]
I'm good with giving this a try. We can always opt out if it gets too spammy.
For UX specific issues see https://github.com/netlify/netlify-cms/projects/9

**Comment by erezrokah** [comment]
Closing this, thank you everyone
